### DISCUSSION TITLE
Custom y axis label width in Bar Chart

### DISCUSSION THREAD

**Discussion by iin2022** [Budibase/budibase#12092] [Help]
Can we set a custom width form y-axis label in Bar Chart ? in my case when it set to horizontal the y-axis label width is too short

**Comment by ConorWebb96** [comment]
Hey @iin2022,

Can you provide a screenshot of what you mean? Possibly a screenshot of your bar chart settings just so I can replicate as well.

Thanks in advance!

**Reply by iin2022** [reply]
Hi Conor,

Please apologize for not giving clear explanation regarding the issue, actually its label column for bar chart, where in my case when I set the bar chart to horizontal, the label column data will be trimmed, so I wonder if we can set the label column width to make the entire label caption can be showed in the chart.

Here is the screenshot :
![image](https://github.com/Budibase/budibase/assets/103981504/73560057-ff9c-447a-8a5c-a0bb6dd33dc6)

Thank you.
